lib/client.js

```diff
@@ -298,6 +298,16 @@ Client.prototype.download = function(src, dest, callback) {
   });
 };
 
+// Tests if the config is correct
+// callback(err, isOk/*true/false*/)
+Client.prototype.testConnection = function(callback) {
```

Comment:
That was what I actually did after. But I stay think this method would be useful.